packages/hyperion-autologging/src/ALInteractableDOMElement.ts

```diff
@@ -452,7 +452,14 @@ function getElementName(element: HTMLElement, surface: string | null, results: A
      */
     if (element.id) {
       try {
-        const labels = document.querySelectorAll<HTMLLabelElement>(`label[for='${element.id}']`);
+        // escape characters that may break the syntax of CSS selector
+        const sanitizedId = element.id.replace(/['"\[\]\(\)]/g, m => {
```

Comment:
Is https://developer.mozilla.org/en-US/docs/Web/API/CSS/escape_static available to use or would it cover the necessary cases here?  Looks like it is specifically for use with things such as `querySelector*`

Reply:
Oh cool. I was looking for this, but landed on https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/escape which is different and deprecated. 
I'll use the one you listed.

Reply:
one update, I noticed jest env does not have the CSS static object. So, I added logic to choose the implementation depending on availability.